libs/sdk/src/transport/mcp-handlers/call-tool-request.handler.ts

```
@@ -1,21 +1,19 @@
-      return result;
-    }
-  } satisfies McpHandler<CallToolRequest, CallToolResult>
-
+    handler: async (request: CallToolRequest, ctx) => {
+      try {
+        const result = await scope.runFlowForOutput('tools:call-tool', { request, ctx });
+        console.log(result);
+        return result;
+      } catch (e) {
+        throw e
+      }
```

ESLint

[error] Unnecessary try/catch wrapper.

(no-useless-catch)